app/services/gov_one_auth_service.rb

```diff
@@ -0,0 +1,95 @@
+class GovOneAuthService
+  # @param code [String]
+  def initialize(code)
+    @code = code
+  end
+
+  # @return [Hash]
+  def tokens
+    body = {
+      grant_type: 'authorization_code',
+      code: @code,
+      redirect_uri: ENV['GOV_ONE_REDIRECT_URI'],
```

Comment:
New ENVs will want to be added to our example file which records all of them in one place. I believe this one should also be set for the dev composition, is that right?

Reply:
[a83da4e](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/a83da4e7325fda7d29d4f24c8abac57759bdedd0) I added the env variables to the example